worlds/pokemon_rb/docs/setup_es.md

```diff
@@ -27,31 +27,31 @@ Una vez que Bizhawk se haya instalado, abre Emuhawk y cambia las siguientes conf
 - (≤ 2.8) Abrir EmuHawk e ir a Config > Customize. Abrir la pestaña Advanced, y en la opción de Lua Core cambiar desde
   "NLua+KopiLua" a "Lua+LuaInterface". Luego reinicia EmuHawk. Esto es fundamental para que el script de Lua funcione
   correctamente.
-  **NOTA: Incluso si "Lua+LuaInterface" ya estaba seleccionado, cambia entre las opciones y vuelvelo a seleccionar.
-  **Algunas instalaciones de versiones nuevas de EmuHawk tienen una tendencia a mostrar "Lua+LuaInterface" por defecto
+  **NOTA: Incluso si "Lua+LuaInterface" ya estaba seleccionado, cambia entre las opciones y vuélvelo a seleccionar.**
+  **Algunas instalaciones de versiones nuevas de EmuHawk tienen una tendencia a mostrar "Lua+LuaInterface" por defecto**
   **pero siguen cargando "NLua+KopiLua" hasta completar este paso.**
```

Comment:
```suggestion
  **NOTA: Incluso si "Lua+LuaInterface" ya estaba seleccionado, cambia entre las opciones y vuélvelo a seleccionar.
  Algunas instalaciones de versiones nuevas de EmuHawk tienen una tendencia a mostrar "Lua+LuaInterface" por defecto
  pero siguen cargando "NLua+KopiLua" hasta completar este paso.**
```

This will do the same thing without needing the bold markdown on each line.

Reply:
if that works, sure, I really just copied the format from the english guide (it's put in each line there)
thanks!